docs/getting-started/index.md

```diff
@@ -1,603 +1,240 @@
 ---
-title: First Steps with JavaScript
-page_title: Kendo UI for Vue Components Introduction - Kendo UI for Vue Docs & Demos
-description: "Get started with the Kendo UI for Vue Native Components using Vite and Composition API."
+title: First Steps (Composition API and JavaScript)
+page_title: Get Started with Kendo UI for Vue - Kendo UI for Vue Docs & Demos
+description: "Get started with the Kendo UI for Vue Native Components and build your first Data Grid by using Vite and the Composition API."
 slug: getting_started_javascript_composition_api
 brand: getting-started
 heading: Get Started
 position: 0
 ---
 
-# Get Started with Kendo UI for Vue Native Components with JavaScript and Composition API
+# Get Started with Kendo UI for Vue
```

Comment:
Not sure it is a good idea that we rely on the following to help people understand what type of Getting started article are they reading.

```
>Curious about TypeScript or the Options API? This tutorial comes in several additional variants:
>* [Kendo UI for Vue with TypeScript and the Composition API](slug:getting_started_typescript_composition_api)
>* [Kendo UI for Vue with TypeScript and the Options API](slug:getting_started_typescript_options_api)
>* [Kendo UI for Vue with JavaScript and the Options API](slug:getting_started_javascript_options_api)
```

Reply:
This was one of my main concerns and I am still not sure. However, here are my thoughts about it:
- This is the main GS article with more than 70% of all page views in the entire Getting Started section. 
- The next article by popularity in the GS section is the GS with TS and Composition API with mere 7% of all page views for this section.
- The data above clearly shows that GS with JS and Composition API is the default article for most people who visit the GS section.
- Adding JS and Composition API to the title simply makes it too long.

So, I figured that if the majority of the people interested in Getting Started with Vue are good with this article as a default approach, then we don't need to state it explicitly in the title. Furthermore:
- The stack is explained in the first sentence and in the side navigation. The note adds even more clarity that we cover other paths.
- It doesn't look like our competitors offer such a wide variety of GS articles, they are good with JS and Composition API.
- After the migration to the new docs platform, we will have code snippet tabs and probably even content tabs that will allow us to combine everything in a single article.

These is my reasoning to drop the stack from the title, but I'm ready to add it back if needed.

Reply:
Shouldn't we also add the following note:

> Historically, all Kendo UI for Vue Native components have supported both **Vue 2** and **Vue 3**. However, Kendo UI for Vue versions released after **November 2024** will no longer support Vue 2. For more information, see [Vue 2 End of Life](https://www.telerik.com/kendo-vue-ui/components/vue2-deprecation/).

Reply:
> Shouldn't we also add the following note:
> 
> > Historically, all Kendo UI for Vue Native components have supported both **Vue 2** and **Vue 3**. However, Kendo UI for Vue versions released after **November 2024** will no longer support Vue 2. For more information, see [Vue 2 End of Life](https://www.telerik.com/kendo-vue-ui/components/vue2-deprecation/).

Getting Started tutorials are usually being consumed by clients that start fresh and look for the latest features in a product. For such clients, this note wouldn't make a lot of sense. If you insist on having the note, add it toward the end of the article so it doesn't dilute the focus of the article.